dashboard_description_plugin/scripts/refresh-md.js

```diff
@@ -9,7 +9,7 @@ dotenv.config({ path: path.resolve(__dirname, "../.env.secrets") });
 
 const workspace = process.env.WORKSPACE;
 const hostname = process.env.BACKEND_URL;
-const backend = "tiger";
+const backend = process.env.BACKEND_TYPE || 'bear';
```

Comment:
Could we please make "tiger" a default option?

Reply:
Sure, I'll change it

Reply:
@andriichumak changed in https://github.com/gooddata/gooddata-plugin-examples/compare/973348080463185479653b6051b911c299044742..fbc943ef299d1d36c145d642099477acdd4447d6